src/templates/tutorial.tsx

```diff
@@ -231,7 +235,8 @@ export const query = graphql`
         sourceUrl
         skill
         published
-        sidebar
+        hideTableOfContents
```

Comment:
Do we want the ability to hide tableOfContents on tutorials and docs?

If not, we can remove this.